web/src/views/IntegrationsView.jsx

```diff
@@ -65,6 +79,12 @@ export default function IntegrationsView() {
       </Col>
       <Col>
         <Title>Server Integrations</Title>
+        <Button
+          onClick={() => createIntegration(accessToken, { name: new Date() })}
```

Comment:
I think it would be better to let the user to decide on the name of the integration themselves, we could also use the name of the integration in the Dashboard.

Reply:
This is an example. We will have much complicated structures based on integrations ( Wix, WooCommerce, Shopify etc.)

Reply:
Ok, I'm approving the PR then.